.github/tests/spire-oidc-insecure/pre-install.sh

```diff
@@ -1,4 +1,10 @@
 #!/usr/bin/env bash
 
-helm install ingress-nginx ingress-nginx --version 4.5.2 --repo https://kubernetes.github.io/ingress-nginx -n "$scenario" --set controller.extraArgs.enable-ssl-passthrough=
+SCRIPT=$(readlink -f "$0")
+SCRIPTPATH=$(dirname "$SCRIPT")
+
+source "$SCRIPTPATH/../repos.sh"
```

Comment:
You can do the sourcing at workflow level in one of the first steps. Then any following step just had those vars available so we don't have to source in each script.

    cat repos.sh >> $GITHUB_ENVIRONMENT

Reply:
What does cating the repo.sh to >> $GITHUB_ENVIRONMENT do? I haven't' found an example of that so far.

Reply:
Oh it creates those environment variables within the workflow.